src/styles/optionRowStyles/index.native.ts

```diff
@@ -7,7 +6,7 @@ import CompactContentContainerStyles from './types';
  *  keeping compactContentContainerStyles as it is.
  *  https://github.com/Expensify/App/issues/14148
  */
-const compactContentContainerStyles: CompactContentContainerStyles = styles.alignItemsCenter;
+const compactContentContainerStyles: CompactContentContainerStyles = (styles) => styles.alignItemsCenter;
```

Comment:
i think this line should be typed

https://github.com/Expensify/App/blob/679d770c439bc26e4d8877c3f556fb26d33fa36e/src/styles/styles.ts#L67